tron/core/actionrun.py

```diff
@@ -599,13 +601,16 @@ def _exit_unsuccessful(self, exit_status=None, retry_original_command=True) -> O
         if self.last_attempt is not None:
             self.last_attempt.exit(exit_status)
         if self.retries_remaining is not None:
-            if self.retries_remaining > 0:
-                self.retries_remaining -= 1
-                return self.restart(original_command=retry_original_command)
+            if exit_status in non_retryable_exit_codes:
+                log.info(f"{self} skipping auto-retries due to non-retryable exit code was recieved.")
```

Comment:
not a blocker, extremely minor rewording for readability
```suggestion
                log.info(f"{self} skipping auto-retries, received non-retryable exit code ({exit_status}).")
```